java/test/jmri/jmrit/operations/locations/gui/LocationEditFrameTest.java

```diff
@@ -737,6 +737,17 @@ public void testSpurTable() {
         Assert.assertEquals("Confirm number of columns", 18, tbl.getColumnCount());
         Assert.assertEquals("Column exists", -1, tbl.findColumn(Bundle.getMessage("DisableLoadChange")));
 
+        // test quick load change
+        track.setQuickLoadChangeEnabled(true);
+        Assert.assertEquals("Confirm number of columns", 19, tbl.getColumnCount());
+        Assert.assertEquals("Column exists", 13, tbl.findColumn(Bundle.getMessage("QuickLoadChange")));
+        tbl.clickOnCell(0, tbl.findColumn(Bundle.getMessage("QuickLoadChange")));
+        Assert.assertFalse("quick load change", track.isQuickLoadChangeEnabled());
+
+        // column should disappear
+        Assert.assertEquals("Confirm number of columns", 18, tbl.getColumnCount());
+        Assert.assertEquals("Column exists", -1, tbl.findColumn(Bundle.getMessage("QuickLoadChange")));
```

Comment:
No need to change, but if -1 means that the column does not exists, then it would seem better to have the text "Column does not exist".

Reply:
Just sloppy on my part.  Fixed.  Thanks